test/Microsoft.ML.Tests/ImagesTests.cs

```diff
@@ -19,6 +20,9 @@ namespace Microsoft.ML.Tests
 {
     public class ImageTests : TestDataPipeBase
     {
+        private static readonly Lazy<bool> _isNotArm = new(() => RuntimeInformation.ProcessArchitecture != Architecture.Arm && RuntimeInformation.ProcessArchitecture != Architecture.Arm64);
```

Comment:
why lazy?

Reply:
Honestly, not sure. I just followed what the example you linked did and they did that way too. Want me to try changing that?

Reply:
Having the following is good enough. This is a test and not worth it to cache the value.

```C#
private static bool IsNotArma => RuntimeInformation.ProcessArchitecture != Architecture.Arm && RuntimeInformation.ProcessArchitecture != Architecture.Arm64;
```